### PR TITLE
Disabled Document Retrieve redaction Engine Error testcases

### PR DESCRIPTION
**Pull request by TabassumJafri**
@mhpnguyen and @sovanncgi

### CODE REVIEW

File: Product/SoapUI_Test/RegressionSuite/Bimodal/AdapterRegistryErrorListMissingTest/AdapterRegistryErrorListMissingTest-soapui-project.xml [hunk 1]
```diff
@@ -1,9 +1,3872 @@
...
+<con:soapui-project abortOnError="false" name="AdapterRegistryErrorListMissingTest" resourceRoot="${projectDir}" runType="SEQUENTIAL" soapui-version="4.5.1" activeEnvironment="Default" id="5c8ba7dc-97c2-41a6-9146-1c1e8ecafc00" xmlns:con="http://eviware.com/soapui/config">
   <con:settings/>
   <con:interface anonymous="optional" bindingName="{urn:gov:hhs:fha:nhinc:entitydocretrieve}EntityDocRetrieveBindingSoap" definition="../../../target/wsdl/EntityDocRetrieve.wsdl" name="EntityDocRetrieveBindingSoap" soapVersion="1_2" type="wsdl" wsaVersion="NONE" xsi:type="con:WsdlInterface" id="2719f138-2b9b-4d4f-b160-05c6702128b0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <con:settings/>
-    <con:endpoints>
+    <con:definitionCache type="TEXT" rootPart="file:/C:/CONNECT_Source/CONNECT/Product/SoapUI_Test/target/wsdl/EntityDocRetrieve.wsdl"><con:part><con:url>file:/C:/CONNECT_Source/CONNECT/Product/SoapUI_Test/target/wsdl/EntityDocRetrieve.wsdl</con:url><con:content><![CDATA[<!--Entity Document Retrieve-->
```
Comment (mhpnguyen) [inline Product/SoapUI_Test/RegressionSuite/Bimodal/AdapterRegistryErrorListMissingTest/AdapterRegistryErrorListMissingTest-soapui-project.xml:6]: Remove definitionCache block

File: Product/SoapUI_Test/RegressionSuite/Bimodal/AdapterRegistryErrorListMissingTest/AdapterRegistryErrorListMissingTest-soapui-project.xml [hunk 2]
```diff
@@ -13,7 +3876,4787 @@
   </con:interface>
   <con:interface xsi:type="con:WsdlInterface" wsaVersion="NONE" name="AdapterDocRetrieveBindingSoap" type="wsdl" bindingName="{urn:gov:hhs:fha:nhinc:adapterdocretrieve}AdapterDocRetrieveBindingSoap" soapVersion="1_2" anonymous="optional" definition="../../target/wsdl/AdapterDocRetrieve.wsdl" id="4151f762-cce3-47eb-8326-d611248b47ea" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <con:settings/>
-    <con:endpoints>
+    <con:definitionCache type="TEXT" rootPart="file:/C:/CONNECT_Source/CONNECT/Product/SoapUI_Test/RegressionSuite/target/wsdl/AdapterDocRetrieve.wsdl"><con:part><con:url>file:/C:/CONNECT_Source/CONNECT/Product/SoapUI_Test/RegressionSuite/target/wsdl/AdapterDocRetrieve.wsdl</con:url><con:content><![CDATA[<!--Adapter Document Retrieve-->
```
Comment (mhpnguyen) [inline Product/SoapUI_Test/RegressionSuite/Bimodal/AdapterRegistryErrorListMissingTest/AdapterRegistryErrorListMissingTest-soapui-project.xml:3879]: Remove definitionCache  block

**Review by mhpnguyen — CHANGES_REQUESTED** [review]
Consider to revert this file and only disable "<con:testCase" block

**Comment by TabassumJafri** [comment]
Closing this PR because push contains the whole soap ui file and it is getting treated as a new file.

**Pull request closed without merging**